src/utils/cloning.ts

```diff
@@ -410,13 +411,16 @@ function cloneASTNodeImpl<T extends ASTNode>(
       [...node.vOverrides].map((o) => cloneASTNodeImpl(o, ast, remappedIds)),
       node.raw,
     );
+  } else if (node instanceof PlaceholderStatement) {
```

Comment:
For ease of navigation the if-else chain is broken up into typed groups, this should go under Statements, in alphabetical order with the other statement types